canvas_modules/common-canvas/src/common-properties/controls/toggletext/toggletext.jsx

```diff
@@ -50,7 +50,7 @@ class ToggletextControl extends React.Component {
 		}
 		let icon = null;
 		if (typeof this.iconsMap[this.props.value] !== "undefined") {
-			icon = <img className="icon" src={this.iconsMap[this.props.value]} />;
+			icon = <img className="icon" src={this.iconsMap[this.props.value]} alt={`${this.props.value} Icon`} />;
```

Comment:
I don't think we can have `Icon` in the text since this wouldn't get translated.  Is the value enough?

Reply:
I added word `Icon` to differentiate between icon and text after icon. 
![Screen Shot 2021-01-04 at 1 02 26 PM](https://user-images.githubusercontent.com/25124000/103579590-65c9a080-4e8d-11eb-8c6f-412fcc993db0.png)

Screen reader reads following for toggle text buttons -
- On Icon On, button, toggletext, Group

If I remove `Icon`, screen reader reads - 
- On On, button, toggletext, Group

Reply:
ok, in that case we'll need to use the proper formatted string.  That way for other languages the value can be moved before or after the `Icon` text.